src/ore/src/lib.rs

```diff
@@ -67,3 +67,7 @@ pub mod str;
 pub mod test;
 pub mod thread;
 pub mod vec;
+
+#[cfg(feature = "tokio_spawn")]
+#[doc(hidden)]
+pub mod tokio;
```

Comment:
Let's call this module `task` rather than `tokio`. (For stdlib/Tokio modules, `ore` prefers to use the module name that the stdlib/Tokio uses.) Can you also sort this into the right spot in the list above?